examples/okta_app_oauth/custom_attributes.tf

```diff
@@ -0,0 +1,16 @@
+resource "okta_app_oauth" "test" {
+  label                      = "testAcc_replace_with_uuid"
+  type                       = "web"
+  grant_types                = ["authorization_code"]
+  redirect_uris              = ["http://d.com/"]
+  response_types             = ["code"]
+  client_basic_secret        = "something_from_somewhere"
+  custom_client_id           = "something_from_somewhere"
+  token_endpoint_auth_method = "client_secret_basic"
+  
+  custom_profile_attributes  = <<JSON
+  {
+    "customAttribute123": "testing-custom-attribute"
```

Comment:
Could we have one of these examples implement a more real world scenario? For instance, the group whitelist scenario you mentioned?

Reply:
I have done this and included comments at the top of the file giving some context with a link to Okta's documentation. I'm not sure how advisable this is as the link may change/disappear.